apps/pie-microsite/package.json

```diff
@@ -11,21 +11,26 @@
     "build": "DEPLOYMENT_ENVIRONMENT='production' eleventy",
     "build:dev": "DEPLOYMENT_ENVIRONMENT='development' eleventy",
     "clean:dist": "run -T rimraf dist",
+    "test": "run -T jest",
```

Comment:
This should tell yarn that the dependency (such as jest) lives in the root-level node-modules.

Otherwise you get the "command 'jest' not found" issue 👍